client/app/components/onboardingGuide/GuideStatusPage.js

```diff
@@ -85,13 +108,10 @@ GuideStatusPage.propTypes = {
   initialPath: string.isRequired,
   name: string.isRequired,
   infoIcon: string.isRequired,
-  nextStep: oneOf([
-    shape({
-      title: string.isRequired,
-      link: string.isRequired,
-    }),
-    bool.isRequired,
-  ]).isRequired,
+  nextStep: shape({
+    title: string.isRequired,
+    link: string.isRequired,
+  }),
```

Comment:
Previously the function that returns the `nextStep` returned either a shape of `false`. So the `oneOf [ shape or bool ]` was set to be required. Now I change the function so that it returns either shape or `null`.